consensus/module.go

```diff
@@ -76,6 +76,40 @@ type consensusModule struct {
 	messagePool map[typesCons.HotstuffStep][]*typesCons.HotstuffMessage
 }
 
+// Functions exposed by the debug interface should only be used for testing puposes.
+type ConsensusDebugModule interface {
```

Comment:
@gokutheengineer Can you add a TODO next to `HandleDebugMessage` in `shared/modules/consensus_module.go` to move it into a `Debug` module in the future?

@deblasis Check out this pattern Gohkan put together. For functions that we want to expose for testing / debugging, but keep it explicitly clear (to the developers)  that it's not part of  as not part of the main interface, we create this Debug module. I'm a fan